src/PKSim.Core/Model/PKSimProject.cs

```diff
@@ -38,6 +39,13 @@ public override bool HasChanged
       /// </summary>
       public virtual IReadOnlyCollection<ISimulationComparison> AllSimulationComparisons => _simulationComparisons;
 
+
+      /// <summary>
+      ///    Returns all <see cref="QualificationPlan" /> defined in the project
+      /// </summary>
+      public virtual IReadOnlyCollection<QualificationPlan> AllQualificationPlans => _qualificationPlans;   
```

Comment:
A project has a bunch of qualification plan. Each plan has a bunch of steps that will be performed one after the other